inc/config_store.h

```diff
@@ -5,220 +5,232 @@
 #include <string.h>
 
 #ifdef __cplusplus
-extern "C" {
+extern "C"
```

Comment:
Did you reformat this doc after including the .clang-format file?
You may need to close and start VS Code again for it to pick the new style.

Reply:
Resolved